plugins/auth-backend/src/identity/DatabaseKeyStore.test.ts

```diff
@@ -18,6 +18,15 @@ import Knex from 'knex';
 import { DatabaseKeyStore } from './DatabaseKeyStore';
 import { DateTime } from 'luxon';
 
+function createDatabaseManager(skipMigrations: boolean = false) {
+  return {
+    getClient: async () => createDB(),
```

Comment:
Technically this could happen to create several new databases by accident, if the underlying code calls getClient more than once. That would make the test fail weirdly. Would be better to just fold this into createDB instead, creating the knex instance once and returning that constant over and over again every time getClient is called.

Reply:
@freben thanks for the review. I changed the code to avoid this situation you described. Could you check again please?